test/unit/components/UI/Countdown.spec.js

```diff
@@ -5,7 +5,7 @@ import { Countdown } from 'components/UI'
 
 describe('component.UI.Countdown', () => {
   it('should render correctly with default props', () => {
-    const wrapper = shallow(<Countdown />)
+    const wrapper = shallow(<Countdown date={new Date(0)} />)
```

Comment:
We should use a date other than the epoch for this. It would make it easier to identify date related issues with the snapshot in the future. I'd suggest simply `new Date()` rather than `new Date(0)`